editor/src/utils/react-performance.spec.ts

```diff
@@ -46,12 +46,12 @@ describe('keepDeepReferenceEqualityIfPossible', () => {
     expect(a[1] === r[1]).toBeTruthy()
   })
 
-  it('keeps reference equality for matching parts of arrays even if they are different in length', () => {
+  it('keeps reference equality for matching parts of arrays even if they are different in length 2', () => {
     const a = deepFreeze([{ a: 5 }, { b: { c: { d: 6 } } }]) as any[]
     const b = deepFreeze([{ a: 5 }]) as any[]
     const r = keepDeepReferenceEqualityIfPossible(a, b)
     expect(fastDeepEquals(b, r)).toBeTruthy()
-    expect(r.length === 1).toBeTruthy
+    expect(r.length === 1).toBeTruthy()
```

Comment:
omg

Reply:
to be honest, I should have rewritten it to be 
```suggestion
    expect(r).toHaveLength(1)
```